hacep/src/main/java/org/kie/hacep/core/infra/utils/SnapshotOnDemandUtils.java

```diff
@@ -140,9 +140,9 @@ private static KafkaConsumer getConfiguredSnapshotConsumer(EnvConfig envConfig)
 
         if (partitionsInfo != null) {
             for (PartitionInfo partition : partitionsInfo) {
-                if (partitions == null || partitions.contains(partition.partition())) {
-                    partitionCollection.add(new TopicPartition(partition.topic(),
-                                                               partition.partition()));
+                TopicPartition topicPartition = new TopicPartition(partition.topic(), partition.partition());
```

Comment:
Seems as not related to documentation refactoring. Please create a separate PR with description of purpose for this change.

Reply:
Ok